### PR TITLE
Added ice_bin handle_shutdown cb

### PR DESCRIPTION
**Pull request by Qizot**
Closes https://github.com/membraneframework/membrane_demo/issues/64

### CODE REVIEW

File: lib/membrane_ice_plugin/ice_bin.ex [hunk 1]
```diff
@@ -243,4 +243,9 @@ defmodule Membrane.ICE.Bin do
 
   @impl true
   def handle_other(msg, _ctx, state), do: {{:ok, notify: msg}, state}
+
+  @impl true
+  def handle_shutdown(_reason, %{connector: connector}) do
+    GenServer.stop(connector)
+  end
```
Comment (mickel8) [inline lib/membrane_ice_plugin/ice_bin.ex:250]: Shouldn't we also implement `terminate` for Connector and kill there ExLibnice?

File: lib/membrane_ice_plugin/connector.ex [hunk 2]
```diff
@@ -99,9 +99,9 @@ defmodule Membrane.ICE.Connector do
     GenServer.call(pid, :restart_stream)
   end
 
-  @spec stop(connector :: pid()) :: :ok
-  def stop(pid) do
-    GenServer.call(pid, :stop)
+  @spec clean_ice(connector :: pid()) :: :ok
```
Comment (mickel8) [inline lib/membrane_ice_plugin/connector.ex:102]: Maybe `reset`? We would have `Connector.run` and `Connector.reset`

File: lib/membrane_ice_plugin/ice_bin.ex (hunk 1 above)
Comment (Qizot) [reply lib/membrane_ice_plugin/ice_bin.ex:250]: I believe that we already have `terminate` callback for connector.

Comment (mickel8) [reply lib/membrane_ice_plugin/ice_bin.ex:250]: Right, we have added it in the previous PR :man_facepalming:

**Review by mickel8 — APPROVED** [review]

**Pull request merged**